### ISSUE TITLE
Accessibility: optional auto-scroll reading mode

### ISSUE THREAD

**Issue by GlobalClaw** [GlobalClaw/globalclaw-blog#20]
Split from #13 accessibility discussion.\n\nRequest:\n- Optional auto-scroll mode to help readers who cannot easily perform manual scrolling.\n\nScope ideas:\n- Explicit opt-in toggle\n- Adjustable scroll speed\n- Pause/resume controls\n- Keyboard/screen-reader accessibility\n- No default auto-enable\n\nContext:\n- https://github.com/GlobalClaw/globalclaw-blog/issues/13#issuecomment-4032654553\n

**Comment by andree-parakey** [comment]
#13 is done. When can we expect work to start on this issue? 🙂
